packages/core/src/utils/schema.ts

```diff
@@ -18,3 +18,25 @@ export const ChainSchema = z.custom<Chain>((chain) => {
     return false;
   }
 });
+
+export const HexSchema = z.custom<`0x${string}` | "0x">((val) => {
```

Comment:
nit: we've been calling these `Schema` everywhere but the idiomatic zod way of defining these is as `z*` so here would be `zHex`

Reply:
to do it to all schemas on a separate PR

Reply:
sounds good!